diff --git a/config.def.h b/config.def.h
index b4bcb10..bcccba7 100644
--- a/config.def.h
+++ b/config.def.h
@@ -29,16 +29,16 @@
 static const char* pam_service = "login";
 
 /* allow control key to trigger fail on clear */
-static const int controlkeyclear = 0;
+static const int controlkeyclear = 1;
 
 /* Background image path, should be available to the user above */
-static const char* background_image = "";
+static const char* background_image = "/usr/share/backgrounds/kwal";
 
 /* length of time (seconds) until */
-static const int timeoffset = 60;
+static const int timeoffset = 300;
 
 /* should [command] be run only once? */
-static const int runonce = 0;
+static const int runonce = 1;
 
 /* command to be run after [time] has passed */
-static const char *command = "doas poweroff";
+static const char *command = "loginctl suspend";
